src/scripts.js

```diff
@@ -24,11 +29,17 @@ let searchInput = document.querySelector("#search-input");
 let showPantryRecipes = document.querySelector(".show-pantry-recipes-btn");
 let tagList = document.querySelector(".tag-list");
 let user;
+let users;
+let ingredientsData;
+let recipeData;
 
 
-window.addEventListener("load", createCards);
-window.addEventListener("load", findTags);
-window.addEventListener("load", generateUser);
+// window.addEventListener("load", createCards);
+// window.addEventListener("load", findTags);
+window.addEventListener("load", getUsers);
+//window.addEventListener("load", generateUser);
+window.addEventListener("load", getIngredients);
```

Comment:
We discussed putting these on load functions through an event handler function as well. Thinking this will come in later with our refactor down the road.

Reply:
Yeah, we sure did. I think it will definitely run better when refactored in that way.